### PR TITLE
Kostiuchenko Maryna 

### PR DESCRIPTION
**Pull request by Marina668**

### CODE REVIEW

**Review by Dmitriy-NK — COMMENTED** [review]
Во всех confusion matrix метки(подписи heatmap) не соответствуют значениям, по умолчанию sklearn.metrics.confusion_matrix считает метку 0 - положительной, так собственный порядок можно задать в аргументе labels, [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)

**Review by Dmitriy-NK — APPROVED** [review]

**Review by Dmitriy-NK — APPROVED** [review]
Looks good.
